pom.xml

```diff
@@ -93,7 +94,7 @@
 
     <slf4j.version>1.7.16</slf4j.version>
     <log4j.version>1.2.17</log4j.version>
-    <log4j2.version>2.13.3</log4j2.version>
+    <log4j2.version>2.12.1</log4j2.version>
```

Comment:
why you lower the log4j2 version?

Reply:
We were struggling with this issue in our logs: https://issues.apache.org/jira/browse/LOG4J2-2901
The logs were not printed on the console and got this error:
`ERROR StatusLogger Reconfiguration failed: No configuration found for '2aae9190' at 'null' in 'null'`

Reply:
@eskabetxe I fixed the problem, and there is no need for the version downgrade :) new commit on its way! :rocket: